registry/rpc/rpcregistry.go

```diff
@@ -183,6 +239,15 @@ func (r *RPCRegistry) SaveUnitState(unitName string, unitState *unit.UnitState,
 		defer debug.Exit_(debug.Enter_(unitName, unitState))
 	}
 
+	r.nocacheSaveUnitState(unitName, unitState, ttl)
+	r.cacheUnitState(unitName, unitState, ttl)
+}
+
+func (r *RPCRegistry) nocacheSaveUnitState(unitName string, unitState *unit.UnitState, ttl time.Duration) {
+	if DebugRPCRegistry {
+		defer debug.Exit_(debug.Enter_(unitName, unitState))
+	}
+
 	if unitState.UnitName == "" {
```

Comment:
okay github was fooling me... I can see the code now. But you need to rebase against the master to see better the diff